bin/react-scripts.js

```diff
@@ -5,14 +5,17 @@ var args = process.argv.slice(3);
 
 switch (script) {
 case 'build':
-case 'start':
 case 'eject':
+case 'start':
   spawn(
     'node',
     [require.resolve('../scripts/' + script)].concat(args),
     {stdio: 'inherit'}
   );
   break;
+case 'test':
+  require('jest/bin/jest');
```

Comment:
Oh, good question. Jest uses `yargs`, so argv should be forwarded properly, right?